test/metabase/util/stats_test.clj

```diff
@@ -68,18 +69,19 @@
 (deftest anonymous-usage-stats-test
   (with-redefs [email/email-configured? (constantly false)
                 slack/slack-configured? (constantly false)]
-    (let [stats (anonymous-usage-stats)]
-      (doseq [[k expected] {:running_on        :unknown
-                            :check_for_updates true
-                            :site_name         true
-                            :friendly_names    false
-                            :email_configured  false
-                            :slack_configured  false
-                            :sso_configured    false
-                            :has_sample_data   false}]
-        (testing k
-          (is (= expected
-                 (get stats k))))))))
+    (mt/with-temporary-setting-values [site-name "Test"]
+      (let [stats (anonymous-usage-stats)]
+        (doseq [[k expected] {:running_on        :unknown
+                              :check_for_updates true
+                              :site_name         true
```

Comment:
Not sure how this was passing before. `site_name` is false unless the corresponding setting is changed to a non-default value.